pycbc/waveform/compress.py

```diff
@@ -685,7 +720,8 @@ def write_to_hdf(self, fp, template_hash, root=None):
         else:
             root = '%s/'%(root)
         group = '%scompressed_waveforms/%s' %(root, str(template_hash))
-        for param in ['amplitude', 'phase', 'sample_points']:
+        for param in ['amplitude', 'phase', 
+                      'amplitude_freq_points', 'phase_freq_points']:
```

Comment:
Interesting. @soumide1102 less than 160 Mb for the SEOBNRv2 ROM interpolants:
```
(200 interpolants points) * (4 vectors) * (250,000 templates) * (8 bytes) / (10248576 MB/Byte) = 156 MB
```
How much of the 2 GB is the TaylorF2 templates?